projects/cdk/abstract/control.ts

```diff
@@ -98,6 +97,10 @@ export abstract class AbstractTuiControl<T>
         return this.safeNgControlData<boolean>(({disabled}) => disabled, false);
     }
 
+    get interactive(): boolean {
+        return !this.readOnly && !this.disabled;
```

Comment:
Let's use `computedDisabled`, components have ability to override it like radio buttons do.